client/profile-wizard/steps/business-details/flows/selective-bundle/index.js

```diff
@@ -48,39 +49,80 @@ export const filterBusinessExtensions = ( extensionInstallationOptions ) => {
 					extensionInstallationOptions[ key ] &&
 					key !== 'install_extensions'
 			)
-			.map( ( key ) => {
-				// Remove anything after :
-				// Please refer to selective-extensions-bundle/index.js
-				// installableExtensions variable
-				// this is to allow duplicate slugs (Tax & Shipping for example)
-				return key.split( ':' )[ 0 ];
-			} )
+			.map( getPluginSlug )
 			// remove duplicate
 			.filter( ( item, index, arr ) => arr.indexOf( item ) === index )
 	);
 };
 
+const timeFrames = [
+	{ name: '0-2s', max: 2 },
+	{ name: '2-5s', max: 5 },
+	{ name: '5-10s', max: 10 },
+	{ name: '10-15s', max: 15 },
+	{ name: '>15s' },
```

Comment:
Sorry for not mentioning this earlier, but I wonder if these timeframes should be stretched out a bit.  I think 99% of first-time installers are going to be in the 15s+ range. cc @pmcpinto @becdetat

Reply:
I added several more time frames all the way up to `>60s`, I would be quite concerned if we hit the `>60s` often (I could even stop at >30s)